packages/salesforcedx-vscode-core/src/commands/describeMetadata.ts

```diff
@@ -15,22 +15,20 @@ import {
 import * as fs from 'fs';
 import * as path from 'path';
 import { mkdir } from 'shelljs';
-import { SfdxCommandletExecutor } from '../commands/util';
 import { workspaceUtils } from '../util';
+import { SfdxCommandletExecutor } from './util';
 
-export class ForceDescribeMetadataExecutor extends SfdxCommandletExecutor<
-  string
-> {
+export class DescribeMetadataExecutor extends SfdxCommandletExecutor<string> {
   public constructor() {
     super();
   }
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   public build(data: {}): Command {
     return new SfdxCommandBuilder()
-      .withArg('force:mdapi:describemetadata')
+      .withArg('org:list:metadata-types')
       .withJson()
```

Comment:
loglevel flag is now deprecated and is added to the command with .withJson(), but if you add false as the argument for that function it won't add the flag!
```suggestion
      .withJson(false)
```